Makefile.am

```diff
@@ -34,7 +34,7 @@ MAINTAINERCLEANFILES =	Makefile.in config.guess config.h.in config.sub \
 						py-compile m4/* po/Makefile.in.in po/Rules-quot \
 						test-driver
 
-CLEANFILES = *~
+CLEANFILES = *~ $(RPM_BUILD_DIR)/*.rpm $(SRPM_BUILD_DIR)/*.rpm
```

Comment:
I guess we can remove the RESULT_DIR as a whole?

And we can also include the generated tarball? anaconda-*.tar.bz2

Reply:
Also I would rather extend cleanfiles bellow the target that polutes these. So bellow rpms and srpms

CLEANFILES +=  $(RPM_BUILD_DIR)/*.rpm $(SRPM_BUILD_DIR)/*.rpm

Reply:
Done

Reply:
Well, as a mater of fact gnu make is expect to have list of files in `CLEANFILES` variable. When I tried to pass directory there - it failed to remove RPMs.